src/masonite/orm/blueprint/Blueprint.py

```diff
@@ -196,16 +199,33 @@ def unsigned(self, column, length=None, nullable=False):
     def unsigned_integer(self):
         pass
 
+    def foreign(self, *columns):
```

Comment:
This definition looks wrong. It should only accept a column name:

```python
table.foreign('profile_id').references('profile').on('id')
```

Reply:
I put *args because of this statement:

```sql
FOREIGN KEY (product_category, product_id)
      REFERENCES product(category, id)
```
Referencing two columns in the same constraint.

Reply:
Here the reference: https://dev.mysql.com/doc/refman/8.0/en/create-table-foreign-keys.html#foreign-key-restrictions

The columns and table aren't quoted with `, I found it strange.